tensorboard/plugins/hparams/backend_context_test.py

```diff
@@ -46,14 +49,20 @@ class BackendContextTest(tf.test.TestCase):
     maxDiff = None  # pylint: disable=invalid-name
 
     def setUp(self):
-        self._mock_tb_context = mock.create_autospec(base_plugin.TBContext)
+        self._mock_tb_context = base_plugin.TBContext()
         self._mock_multiplexer = mock.create_autospec(
```

Comment:
Ditto here re: having a TODO to clean up the test code here so that if it's going to use mocks it at least mocks the DataProvider APIs directly rather than mocking EventMultiplexer and then wrapping that in MultiplexerDataProvider.

Reply:
Will do.